src/components/templates/PartyMap/components/RoomModal/RoomModal.tsx

```diff
@@ -60,8 +60,8 @@ export const RoomModal: React.FC<RoomModalProps> = ({
       <div className="room-modal">
         <RoomModalContent
           room={room}
-          venueName={venue.name}
           venueEvents={venueEvents}
+          venue={venue}
           showSchedule={venue.showSchedule}
```

Comment:
remove showSchedule from here

Reply:
fixed in 8ad5fbfed8829f5455a0c0db68dfe7facb1dbac8